### ISSUE TITLE
Help wanted: Permission dialog not showing up in pgb

### ISSUE THREAD

**Issue by bigreni** [dpa99c/cordova-diagnostic-plugin#255]
I am using phonegap build. I am trying to get permission to access user location. I have added the geolocation plugin
    <gap:plugin name="cordova-plugin-geolocation" source="npm"/>

I have also added the diagnostic plugin to my config.xml
    <gap:plugin name="cordova.plugins.diagnostic" source="npm" />

This is my code that runs after my deviceready event
   cordova.plugins.diagnostic.getLocationAuthorizationStatus(function (status) {
        switch (status) {
            case cordova.plugins.diagnostic.permissionStatus.NOT_REQUESTED:
                cordova.plugins.diagnostic.requestLocationAuthorization(function (status) {
                }, function (error) {
                    console.error(error);
                });
                break;
            case cordova.plugins.diagnostic.permissionStatus.GRANTED:
                break;
            case cordova.plugins.diagnostic.permissionStatus.DENIED:
                cordova.plugins.diagnostic.requestLocationAuthorization(function (status) {
                }, function (error) {
                    console.error(error);
                });
                break;
            case cordova.plugins.diagnostic.permissionStatus.DENIED_ALWAYS:
                break;
        }
    }, function (error) {
        console.error(error);
    });

I would expect the user to get a popup when they first install the app asking them to grant permission. I have tested on a phone with Android 6 and there is no dialog. Can you please tell what am I doing incorrectly? Thank you for your help.

**Comment by dpa99c** [comment]
Very probably related to #254: 
TL;DR: Latest version of Phonegap Build CLI no longer supports `<config-file>` blocks to insert `<uses-permission>` into `AndroidManifest.xml`.

**Comment by bigreni** [comment]
Thanks for pointing me to the other thread. I got it to work. I am using version pgb version 6.5 which supports <config-file>. I added the following to my config.xml and it works.

xmlns:android="http://schemas.android.com/apk/res/android"
    <gap:config-file platform="android" parent="/manifest" mode="add">
        <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
        <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
    </gap:config-file>

Hopefully this will help other people using pgb.
Thank you again.
